### ISSUE TITLE
ENH: density plot of I(Q) stack

### ISSUE THREAD

**Issue by chiahaoliu** [xpdAcq/xpdView#5]
in additional to waterfall plot

**Comment by CJ-Wright** [comment]
What is a "density plot"?

**Comment by chiahaoliu** [comment]
Suggest by Anton, it's used to summarize a series of 1D data. The x-axis of the density plot is the data grid (Q, r or whatever) and y being the index of 1D data (implies evolvements of experiment) and z is the intensity at data grid x and index y.

**Comment by CJ-Wright** [comment]
Hmm, I don't know if I grok this yet (do you have an example image?) I might not need to though.

**Comment by chiahaoliu** [comment]
Yeah, that's actually not too different from a water flow plot with proper y-offset between curves. I don't have example at hand, but I can show you tomorrow.

**Comment by CJ-Wright** [comment]
ok
